test/server/sockets.js

```diff
@@ -3,6 +3,7 @@
 const assert = require('assert').strict;
 const cluster = require('cluster');
 
+// These tests are failing due to syntax errors, not necessarily failing tests
```

Comment:
This test has never worked, and was not updated to the latest version of Sockets anyway. That's probably what should be noted.